monkey/monkey_island/cc/app.py

```diff
@@ -151,11 +156,16 @@ def init_api_resources(api: FlaskDIWrapper):
     api.add_resource(ManualExploitation, "/api/exploitations/manual")
     api.add_resource(MonkeyExploitation, "/api/exploitations/monkey")
 
+    # API Spec: Why is the endpoint separated? Why not just
+    # "/api/zero-trust-finding-event/<string:finding_id>"?
     api.add_resource(ZeroTrustFindingEvent, "/api/zero-trust/finding-event/<string:finding_id>")
     api.add_resource(TelemetryFeed, "/api/telemetry-feed")
+    # API Spec: What log? Agent log?
     api.add_resource(Log, "/api/log")
+    # API Spec: Why the inconsistency in endpoints of IslandLog and Log?
     api.add_resource(IslandLog, "/api/log/island/download")
 
+    # API Spec: These two should have the same url syntax
```

Comment:
They should probably be the same resource, no? `GET` for download and `POST` for upload.